classes/identity/Identity.php

```diff
@@ -280,6 +280,19 @@ public function getOrcid()
         return $this->getData('orcid');
     }
 
+    /**
+     * Return the string that should be displayed when showing a user's ORCiD
+     *
+     */
+    public function getOrcidDisplayValue(): ?string
+    {
+        if (!$this->getOrcid()) {
+            return null;
+        }
+
+        return $this->hasVerifiedOrcid() ? $this->getOrcid() : $this->getOrcid() . ' (' . __('orcid.unauthenticated') . ')';
```

Comment:
I don't really have a strong opinion on it, but did you consider having the parenthesis as part of the locale string instead of direct formatting here? I don't know if there would be any circumstances where other languages would have issues with this formatting, but just wanted to throw it out there.

Reply:
Don't quite remember why I decided to format it in PHP, but I've found existing strings with parenthesis in the locale  files so I've moved these to the locale string.